docs/adr/adr-005-record-architecture-decisions.md

```diff
@@ -0,0 +1,21 @@
+# 1. Record architecture decisions
+
+Date: 2020-10-21
+
+## Context
+
+While we have already recorded architectural decisions for this project, the structure of these documents has been arbitrary, since there was no guidance to follow. Recently we found that creating a new ADR was [hindered](https://github.com/alphagov/email-alert-api/pull/1441#discussion_r508729384) by wanting to be consistent with the structure of previous ADRs.
+
+## Decision
+
+We will continue to use Architecture Decision Records, as described by Michael Nygard in this article: http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions.
+
+We will adopt a structure for future ADRs that matches this document - the exemplar in [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#documenting-your-decisions). We will not change how the files are named, to avoid breaking links that were never expected to change.
+
+## Status
+
+Accepted
+
+## Consequences
+
+None.
```

Comment:
There's a consequence that the first 4 ADRs are in a different format.

Reply:
Oh yes, good point - I usually read "consequences" as "actions". I've added that in.